src/Splitter/SplitTestsByTimeTask.php

```diff
@@ -57,7 +61,7 @@ public function run(): void
 
         arsort($testsWithTime);
 
-        for ($i = 0; $i < $this->numGroups; $i++) {
+        for ($i = 0; $i < $this->numGroups; ++$i) {
```

Comment:
Why did you change it from post-increment to pre-increment?
This is not the only place :-)

Reply:
Short answer: Performance. Pre-increment is faster than post-increment.
Long answer: See Explanation of why [here](https://stackoverflow.com/a/1756094) and [here](https://piotrzatorski.medium.com/pre-and-post-increment-decrement-how-do-they-work-7319b1ddef1d), or a performance test [here](https://syframework.alwaysdata.net/pre-increment).

Reply:
I don't see that performance improvement does matter here... 
But it is not a big deal. I think this PR can be merged